Altis_Life.Altis/stringtable.xml

```diff
@@ -4478,7 +4478,7 @@
         <Key ID="STR_CouldNotRemoveItemsToPutInVeh">
             <Original>Couldn't remove the items from your inventory to put in the vehicle.</Original>
             <Czech>Vozidlo je buď plný nebo nemůže myslet si, že mnoho.</Czech>
-            <French>Le véhicule est plein ou ne peut pas supporter autant d'item.</French>
+            <French>Impossible de retirer les objets de votre inventaire pour les mettre dans le véhicule.</French>
```

Comment:
Heavier sentence w/ same meaning than the old one.

Reply:
"Supporter autant d'item" for a vehicule is not correct.

Reply:
"Le véhicule est déjà plein ou ne peut pas stocker autant d'objets en plus."

?

Reply:
Correct also.

Reply:
"Le véhicule est déjà plein ou ne peut pas stocker d'avantage d'objets." 
?

Reply:
@Metalhearf same as mine...